src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -1882,6 +1882,31 @@ public void testContainerWithMoreCpuOptions() throws Exception {
   }
 
   @Test
+  public void testContainerWithNanoCpusOptions() throws Exception {
```

Comment:
Could you replace this integration test with a unit test instead? See https://github.com/spotify/docker-client/blob/master/CONTRIBUTING.md#unit-tests-and-integration-tests